### PR TITLE
Completely remove SendFil

### PR DESCRIPTION
**Pull request by asutula**
Had previously removed it from some apis, but not all. Now it's gone gone.

### CODE REVIEW

File: ffs/api/api_wallet.go [hunk 1]
```diff
@@ -81,14 +80,6 @@ func (i *API) VerifyMessage(ctx context.Context, addr string, message, signature
 	return ok, nil
 }
 
-// SendFil sends fil from a managed address to any another address, returns immediately but funds are sent asynchronously.
```
Comment (jsign) [inline ffs/api/api_wallet.go:84]: Do we need to delete it from the internal API? Sounds to me that only at the gRPC level is enough.

Comment (asutula) [reply ffs/api/api_wallet.go:84]: Yea, actually realized that and reversing now. Same in wallet module.

Comment (andrewxhill) [reply ffs/api/api_wallet.go]: doh... this just made me wonder. couldn't it be a flag? seems like those running their own Pow would still want these.

Comment (asutula) [reply ffs/api/api_wallet.go]: hmmmm, I'm not sure. thinking.

**Review by jsign — APPROVED** [review]

**Comment by asutula** [comment]
closing in favor of https://github.com/textileio/powergate/pull/684

**Pull request closed without merging**